label_studio/projects/models.py

```diff
@@ -1200,6 +1200,15 @@ def update_created_annotations_and_labels(self, annotations):
         self.save(update_fields=['created_annotations', 'created_labels'])
 
     def remove_created_annotations_and_labels(self, annotations):
+        # we are going to remove all drafts, make full refresh
+        if self.project.annotations.count() == len(annotations):
+            # it's hard to calculate created_labels_drafts properly, so it's better to reset it completely
```

Comment:
these comments are copy-pasted from below, please fix them to refer to annotations and labels rather than drafts